### DISCUSSION TITLE
Improving the Arabic language

### DISCUSSION THREAD

**Discussion by zefr0x** [organicmaps/organicmaps#1311] [Q&A] [answered]
I found meny errors in the Arabic language so I want to fix them but I have a question, can I use [diacritics](https://wikiless.org/wiki/Arabic_diacritics?lang=en) without having a problem in the sound or the UI for example? Since they are relly important in some words.

I know there should be no problem with the diacritics but because they were never written in the previous translation I thought it was better to ask.

**Comment by biodranik** [comment, marked as answer]
For voice, you need to test diacritics with Text-To-Speech (TTS) engines you have on your device. You should be able to do it without Organic Maps. It only depends on the TTS, not on our code.

For displaying on the map we have #460 and #516 issues that should be fixed.

Interface translations should work fine.
